### ISSUE TITLE
Warning Msg on @phenomnomnominal/tsquery@2.1.0 on cli installation

### ISSUE THREAD

**Issue by marioestradarosa** [strongloop/loopback-next#1769]
## Description / Steps to reproduce / Feature proposal
```sh
npm install -g @loopack/cli
```

## Current Behavior
```sh

npm WARN @phenomnomnominal/tsquery@2.1.0 requires a peer of typescript@^2 but none is installed. You must install peer dependencies yourself.

+ @loopback/cli@0.26.0

```

The following package has declared the typescript as peerDependencies, I ran the commands and work fine, however, this message still appears as a warning.

```sh 
--package.json--
"@phenomnomnominal/tsquery",
  "peerDependencies": {
    "typescript": "^2"
  },
```

## Expected Behavior

Not seeing this message?

_See [Reporting Issues](http://loopback.io/doc/en/contrib/Reporting-issues.html) for more tips on writing good issues_

**Comment by marioestradarosa** [comment]
@virkt25  you are good at these things 😄

**Comment by phenomnomnominal** [comment]
Released `@phenomnomnominal/tsquery@2.1.1`, hopefully that sorts it
